docs/pages/faqs.md

```diff
@@ -2,6 +2,21 @@
 
 This section of the documentation will cover common questions and encounters often asked by users and developers.
 
+## Tooling
+
+:::note "Incorrect yarn version"
+
+Lodestar is using [yarn classic](https://classic.yarnpkg.com/lang/en/). Using more recent versions will lead to issues (e.g. spurious insertion of `packageManager` field in `package.json`).
```

Comment:
What about adding `packageManager` to package.json?

Reply:
Might be an option, not sure if others are fine with that.

Reply:
I think that we should set `packageManager` in package.json and tailor docs around that.